apps/betterangels-backend/shelters/templates/shelters/time_range_widget.html

```diff
@@ -0,0 +1,270 @@
+{% comment %} Render the standard text input {% endcomment %}
```

Comment:
Have you tested this on mobile? On desktop, I can drag and drop to select a range of times, but on mobile I have to tap each one individually. Also, I think not all shelters may have intake times scheduled on the hour or half-hour, there may be shelters which have intervals that end on the 10's or 15's i.e 10:50pm or 10:45pm.